### ISSUE TITLE
Don't show "(unassigned)" in doc for unassigned keyboard shortcuts

### ISSUE THREAD

**Issue by weinand** [Microsoft/vscode-docs#1425]
Like this for debugging:

![2018-02-19_14-46-28](https://user-images.githubusercontent.com/1898161/36380799-da4858c0-1583-11e8-9a4f-9228a6fbea21.png)

It would be great if we could just omit the whole "(unassigned)" in the case that the keyboard shortcut is unassigned.

If that is difficult to do we could at least reword this to "(unassigned keyboard shortcut)" which still looks silly but better than "(unassigned)".

**Comment by gregvanl** [comment]
This is caused by a typo. 'editor.actions.triggerSuggest' should be 'editor.action.triggerSuggest'.
